gef.py

```diff
@@ -8109,6 +8118,38 @@ def __init__(self) -> None:
         return
 
 
+class PatchGefCommand(GenericCommand):
+    """Write specified variable-byte-content to the specified memory location pointed by ADDRESS."""
+
+    _cmdline_ = "patch gef"
+    _syntax_  = f"{_cmdline_} ADDRESS $_gef0"
+    _example_ = f"{_cmdline_} $sp $_gef3"
+
+    @only_if_gdb_running
+    def do_invoke(self, argv: List[str]) -> None:
+        argc = len(argv)
+        if argc != 2:
+            self.usage()
+            return
+
+        location, s = argv[0:2]
```

Comment:
Give meaningful names to variable.

```suggestion
        location, value = argv[0:2]
```